src/Control/Lens/Iso.hs

```diff
@@ -51,16 +51,15 @@ module Control.Lens.Iso
   , enum
   , curried, uncurried
   , flipped
-  , Swapped(..)
+  , swapped
```

Comment:
This is a breaking change, so now would be a good time to bump `lens`'s version number to `4.20`.

Reply:
Good point. I expect the compatibility changes for GHC-9.0 to introduce some other breaking changes too, so I'd rather hold up this PR until then. I'll improve other aspects though now.

Reply:
> I expect the compatibility changes for GHC-9.0 to introduce some other breaking changes too, so I'd rather hold up this PR until then.

I've just landed #942, which adds support for GHC 9.0 and bumps the major version number.

Reply:
Are the breaking change in `TH` modules, as the #942 changelog entries don't highlight what is broken.

Reply:
> Are the breaking change in `TH` modules

That's right.

Reply:
Ok. I'll come back to this ASAP, in a day or two. Lets have more breaking changes :)